src/fasp_to_fenics.cpp

```diff
@@ -82,3 +82,60 @@ dvector EigenVector_to_dvector(const dolfin::EigenVector* vec_b)
   dVec_b.val = (double*) vec_b->data();
   return dVec_b;
 }
+
+/**
+ * \fn EigenVector Copy_dvector_to_EigenVector(dolfin::vector* VecSolu, const dvector* vec_b)
+ *
+ * \brief Copy the dvector vec_b to dolfing::EigenVector
+ *
+ * \param
+ *        VecSolu  dolfin::function (output)
+ *        vec_b    dvector to be converted
+ *
+ * \return         dolinf::vector if conversion successful; otherwise error information.
+ */
+EigenVector Copy_dvector_to_EigenVector(const dvector* vec_b)
```

Comment:
Let's try to keep the namespace on these:
``dolfin::EigenVector Copy_dvector_to_EigenVector(const dvector* vec_b)``